tool/truffle/jruby_truffle_runner/gem_configurations/activesupport.yaml

```diff
@@ -0,0 +1,24 @@
+---
+:global:
+  :jruby_truffle_path: '../../jruby/bin/ruby'
+:setup:
+  :after:
+    - 'echo "$:.unshift \"\#{path}/..\"" >> "#{options[:global][:truffle_bundle_path]}/bundler/setup.rb"'
+  :file:
+    shims.rb: |
+              module Minitest
+                module Assertions
+                  def self.diff
+                    nil
+                  end
+                end
+              end
+    bundler.rb: "module Bundler; def self.setup; end; end"
+    method_source.rb: nil
+  :without:
+    - db
+:run:
+  :require:
+    - shims
+  :load_path:
+    - test
```

Comment:
These are all a bit cryptic to me - can you explain why they're needed for Active Support?